VTUL/iawa
# Add static manifest link for Mirador viewer

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1785) (:star:)

* Other Relevant Links (https://github.com/VTUL/InstallScripts/pull/194)

# What does this Pull Request do? (:star:)
This PR creates a static IIIF manifest link in AWS S3 to be used by Mirador Viewer to display images of an IAWA item. 

# What's the changes? (:star:)

* Changes manifest link to a static link served by AWS S3 static IIIF image server
* Reads manifest link base url from config/secrets.yml
* Refactor batch import so no files will be uploaded or processed through web interface

# How should this be tested?

* This PR is dependent on the the following PR to InstallScripts: VTUL/InstallScripts#194. Please wait until this one has been merged first before using InstallScripts to build the application.
* Update your ansible/site_secrets.yml as in example_secrets.yml to set iiif_manifest_base_url: "https://s3.amazonaws.com/scholar-jekyll.dev.vtlibcloud.com/iiif_s3_test/"
* Once the VM is running, go to the Dashboard. Click on Collections on the left and create a Collection first with title and identifier (Ms1990_057_Chadeayne).
* Click on Works on the left. Click on the "Batch Import Items" button.
* Enter manifest CSV file and corresponding image directory path.
* Go to an item display page, you should see images shown in Mirador viewer which uses the pre-generated manifest link such as "https://s3.amazonaws.com/scholar-jekyll.dev.vtlibcloud.com/iiif_s3_test/Ms1990_057_Chadeayne/Edited/Ms1990_057_Box1/Ms1990_057_B001_F001_001_LHJClips_Ms/manifest.json" 

# Additional Notes:
This PR should be tested in branch LIBTD-1785

# Interested parties
Tag (@yinlinchen ) interested parties

(:star:) Required fields
